src/components/TodoList.js

```diff
@@ -19,7 +19,7 @@ class TodoList extends React.Component {
             todos
          } = this.props;
          return (
-            <ul className="todoList-wrapper">
+            <ul className='todoList-wrapper'>
                 {todos.map((key, value, id) => {
```

Comment:
Just from my feeling you could also filter by empty title here and would not create an empty component at all. It could be ok for List to know what data is relevant to render children.